Why would upstream add the arch name at the end of all binaries?

Index: CMakeLists.txt
--- CMakeLists.txt.orig
+++ CMakeLists.txt
@@ -125,6 +125,9 @@ else()
 	elseif(CMAKE_SYSTEM_PROCESSOR MATCHES "^x86.64$")
 		set(X86 ON)
 		set(Architecture "x86_64")
+	elseif(CMAKE_SYSTEM_PROCESSOR MATCHES "^amd64$")
+		set(X86 ON)
+		set(Architecture "x86_64")
 	elseif(CMAKE_SYSTEM_PROCESSOR STREQUAL "powerpc")
 		set(Architecture "ppc")
 	elseif(CMAKE_SYSTEM_PROCESSOR STREQUAL "powerpc64")
@@ -145,19 +148,19 @@ endif()
 # Binary names
 #
 #=============================================================================
-set(SPEngine "openjk_sp.${Architecture}")
-set(SPGame "jagame${Architecture}")
-set(SPRDVanillaRenderer "rdsp-vanilla_${Architecture}")
-set(MPEngine "openjk.${Architecture}")
-set(MPVanillaRenderer "rd-vanilla_${Architecture}")
-set(MPDed "openjkded.${Architecture}")
-set(MPGame "jampgame${Architecture}")
-set(MPCGame "cgame${Architecture}")
-set(MPUI "ui${Architecture}")
-set(JK2SPEngine "openjo_sp.${Architecture}")
-set(JK2SPGame "jospgame${Architecture}")
-set(JK2SPVanillaRenderer "rdjosp-vanilla_${Architecture}")
-set(AssetsPk3 "openjk-${Architecture}.pk3")
+set(SPEngine "openjk_sp")
+set(SPGame "jagame")
+set(SPRDVanillaRenderer "rdsp-vanilla")
+set(MPEngine "openjk")
+set(MPVanillaRenderer "rd-vanilla")
+set(MPDed "openjkded")
+set(MPGame "jampgame")
+set(MPCGame "cgame")
+set(MPUI "ui")
+set(JK2SPEngine "openjo_sp")
+set(JK2SPGame "jospgame")
+set(JK2SPVanillaRenderer "rdjosp-vanilla")
+set(AssetsPk3 "openjk.pk3")
 # Library names
 set(MPBotLib "botlib")
 set(SharedLib "shared")
@@ -386,4 +389,4 @@ endif()
 if(BuildTests)
 	enable_testing()
 	add_subdirectory("tests")
-endif()
\ No newline at end of file
+endif()
